### PR TITLE
Issue #1071 Update cucumber step commandReturnShouldEqual

### PR DESCRIPTION
**Pull request by agajdosi**
Addresses issue #1071

### CODE REVIEW

**Comment by hferentschik** [comment]
Seems to be breaking integration tests!?

**Comment by agajdosi** [comment]
@hferentschik yes, I've got into some git mistake, need to slow down and resolve it, sorry for that :(

**Comment by hferentschik** [comment]
No worries.

File: test/integration/integration_test.go [hunk 1]
```diff
@@ -123,9 +123,9 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`executing "oc ([^"]*)" (succeeds|fails)$`, minishift.executingOcCommandSucceedsOrFails)
 
 	// steps to verify stdout and stderr of commands executed
-	s.Step(`([^"]*) should contain ([^"]*)$`, commandReturnShouldContain)
+	s.Step(`([^"]*) should contain "([^"]*)"$`, commandReturnShouldContain)
```
Comment (praveenkumar) [inline test/integration/integration_test.go]: We are doing it to make sure we capture exitcode which is part of `"<exitcode>"` regex so if we have something extra can be part of `"<what_ever>"` part of this. Quite neat 👍

File: test/integration/integration_test.go [hunk 2]
```diff
@@ -123,9 +123,9 @@ func FeatureContext(s *godog.Suite) {
...
 	s.Step(`([^"]*) should contain$`, commandReturnShouldContainContent)
-	s.Step(`([^"]*) should equal ([^"]*)$`, commandReturnShouldEqual)
+	s.Step(`([^"]*) should equal "([^"]*)"$`, commandReturnShouldEqual)
```
Comment (budhrg) [inline test/integration/integration_test.go]: :+1:

**Comment by hferentschik** [comment]
Merged, thanks.

**Pull request closed without merging**